core/src/main/java/org/apache/iceberg/TableScanContext.java

```diff
@@ -59,6 +60,11 @@ public boolean returnColumnStats() {
     return false;
   }
 
+  @Value.Default
+  public Collection<Integer> returnColumnStatsToInclude() {
```

Comment:
I would call this `columnsToIncludeStats`. return here is not intuitive here, while it makes sense for the method above.

Reply:
done